src/interfaces/classes.ts

```diff
@@ -38,6 +38,6 @@ export const WithClasses = {
             result.push(...WithClasses.splitClasses(cls))
         }
 
-        return dedup(result.sort((a, b) => a.localeCompare(b)))
+        return dedup(result.filter(c => c.length).sort((a, b) => a.localeCompare(b)))
```

Comment:
Empty classes should never be returned from `WithClasses.splitClasses`. I now see that they may be, but it's a bug (I'm too used to Python, where `' a  b '.split()` is `['a', 'b']` and not `['', 'a', '', 'b', '']`).

The proper fix would probably be to change `const WHITESPACE` to `/[ \t\n]+/`, and `splitClasses` to `yield* str.trim().split(WHITESPACE)`.